pkg/procmatch/cmd/procmatch/procmatch.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/DataDog/procmatch"
```

Comment:
Incorrect import. Do we still need this repo if we're just embedding the library within this agent?